api/bundle/models/bundle.settings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "kind": "collectionType",
+  "collectionName": "bundles",
+  "info": {
+    "name": "Bundle"
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true,
+    "draftAndPublish": true
+  },
+  "attributes": {
+    "Title_EN": {
+      "type": "string",
+      "required": true
+    },
+    "Title_FR": {
+      "type": "string",
+      "required": true
+    },
+    "Description_EN": {
+      "type": "text"
+    },
+    "Description_FR": {
+      "type": "text"
+    },
+    "benefits": {
```

Comment:
It looks like there is an image associated to bundles based off the Figma. I wrote a comment in the design asking Fred/Kevin